src/libpmemobj/obj.c

```diff
@@ -1148,6 +1148,34 @@ pmemobj_create(const char *path, const char *layout, size_t poolsize,
 	return NULL;
 }
 
+#ifdef _WIN32
+/*
+ * pmemobj_createW -- create a transactional memory pool (set)
+ */
+PMEMobjpool *
+pmemobj_createW(const wchar_t *path, const wchar_t *layout, size_t poolsize,
```

Comment:
Why are there free's all of the sudden in this code? I'm not seeing any mallocs.
Unless it is absolutely necessary, a function should never return a pointer to a allocated memory that the caller is responsible for freeing.